core-skills/web-application-development-with-react/README.md

```diff
@@ -118,6 +118,8 @@ The candidate should create a React application which satisfies the following.
   - Candidate should showcase their component to assessor.
   - Candidate should walk assessor through tests which give maintainers confidence
     that the component displays data from an API.
+  - Testing this component should include unit tests that demonstrate the boundary between the view logic and domain logic,
+    and acceptance (end to end) testing that demonstrates data retrieved is shown in the UI.
```

Comment:
```suggestion
    and acceptance (end to end) testing that demonstrates that all components (which are unit tested independently) interact with each other correctly.
```

I'm not sure we should require it to be UI (I can't think of a better way), but if there is then :+1:

Reply:
An end to end automated test is what was _asked_ of me, prompting this update. I don't think that your way is specific enough actually, and a test that shows component interaction but doesn't test UI rendering is moot IMO

Reply:
I'm concerned that we ask someone to do UI testing, but haven't taught someone WHY we're asking this.